redbot/core/commands/context.py

```diff
@@ -211,6 +218,12 @@ async def send_interactive(
                         # or channel is a DM
                         with contextlib.suppress(discord.HTTPException):
                             await query.delete()
+                    if resp.content == "file":
```

Comment:
Currently, input such as `FILE` is treated identically to `more`.

Due to the predicate `lower_contained_in` being used, `resp.content` may contain capitals. This should either inspect the `result` attribute of the predicate to see if it is `1`, or also casefold the content before comparing here.